src/lib/ModelConfig.cc

```diff
@@ -621,6 +621,16 @@ void ModelConfig::validate() {
   }
   subFields.clear();
 
+  // Environment-Variables
+  root = "Environment-Variables";
+  size_t numEnvVars = configFile_[root].size();
+  for (size_t i = 0; i < numEnvVars; i++) {
+    char envVarNum[50];
```

Comment:
why is the hard-coded value 50?

Reply:
This char* is just used to give each environment variable a label so that inside nodeChecker we can validate that each env variable is readable as a string. The constant part of the label-string is 22 chars, so the additional space can guarentee that each label can be assigned a unique ID number (up to 28 digits long, much more than ever will be needed)

Reply:
Comment now irrelevant